matrix-neoboard-widget/Dockerfile

```diff
@@ -6,7 +6,6 @@ ENV REACT_APP_VERSION=${REACT_APP_VERSION}
 ENV REACT_APP_REVISION=${REACT_APP_REVISION}
 
 ADD build /usr/share/nginx/html/
-ADD LICENSE /usr/share/nginx/html/LICENSE.txt
```

Comment:
Shouldn't we keep the licence?

Reply:
This line is not needed, LICENSE is copied by `generate-disclaimer` script.